app/controllers/restaurants_controller.rb

```diff
@@ -14,7 +14,8 @@ def show
   # POST /restaurants
   def create
     # create! RecordInvalid exception handled in ApplicationController
-    Restaurant.create!(restaurant_params)
+    rest = Restaurant.create!(restaurant_params)
```

Comment:
`rest` is it to rest and relax, or shortcut and not explicit naming for `restaurant`? :)